kong.conf.default

```diff
@@ -357,6 +362,17 @@
                                  # To read the file again after modifying it,
                                  # Kong must be reloaded.
 
+#dns_order = last,srv,a,cname    # the order in which to resolve different
+                                 # record types. The `last` type means the
+                                 # type of the last succesful lookup (for the
+                                 # specified name). The format is a (case
+                                 # insensitive) comma separated list.
+
+#dns_not_found_ttl = 30.0        # ttl in seconds for empty dns responses and
+                                 # "(3) name error" responses.
+
+#dns_error_ttl = 1.0             # ttl in seconds for error responses.
```

Comment:
@thibaultcha I fixed the style issues (somehow missed them after the discussion). But not the ttl.

See original PR message above. Timeouts are all in `ms`, but ttl's are consistently in `seconds` in the file. And that does make sense to me.